### PR TITLE
display PR web URL

### PR DESCRIPTION
**Pull request by ligz08**
Display PR's web URL after it's successfully created.

### CODE REVIEW

File: task/createPullRequest.ps1 [hunk 1]
```diff
@@ -538,7 +538,7 @@ function CreateAzureDevOpsPullRequest() {
 
     $jsonBody = ConvertTo-Json $body
     Write-Host $jsonBody
-    $url = "$env:System_TeamFoundationCollectionUri$($teamProject)/_apis/git/repositories/$($repositoryName)/pullrequests?api-version=4.0"
+    $url = "$env:System_TeamFoundationCollectionUri$($teamProject)/_apis/git/repositories/$($repositoryName)/pullrequests?api-version=7.0"
```
Comment (shayki5) [inline task/createPullRequest.ps1:541]: Is the URL not available without upgrade to version 7.0?

Comment (ligz08) [reply task/createPullRequest.ps1:541]: I don't know for sure if 4.x version guarantees including this in response. For me I do get `webUrl` field even when setting `api-version=4.0`, but [API version 4.1's documentation](https://learn.microsoft.com/en-us/rest/api/azure/devops/git/pull-requests/create?view=azure-devops-rest-4.1&tabs=HTTP#gitpullrequest) does not mention `webUrl`, so I don't know if everyone gets the same, thus opted for `api-version=7.0` ([doc](https://learn.microsoft.com/en-us/rest/api/azure/devops/git/pull-requests/create?view=azure-devops-rest-7.0&tabs=HTTP#gitpullrequest)) to play safe.

Comment (shayki5) [reply task/createPullRequest.ps1:541]: OH ok, I just want to keep on backward compatibility, if someone using the extension in TFS 2018 he can't use `api-version=7.0`.
Can you do the test to the URL if it's `dev.azure.com` that is Azure DevOps so use version 7 and print the URL and if not keep it on 4? you can see an example to this in the code.

Comment (ligz08) [reply task/createPullRequest.ps1]: Good idea, added [e4d2b6e](https://github.com/shayki5/azure-devops-create-pr-task/pull/164/commits/e4d2b6e4871f6b71a09b22d5b3db24e699556352).

**Review by shayki5 — APPROVED** [review]

**Pull request merged**